seed_data.sh

```diff
@@ -14,3 +14,4 @@ python manage.py loaddata payment
 python manage.py loaddata order
 python manage.py loaddata order_product
 python manage.py loaddata favoritesellers
+python manage.py loaddata recommendations
```

Comment:
wonderful! Since you've created a `productratings.json` file in your fixtures, you may consider adding a `... loaddata productratings` line here in your `./seed_data.sh` file. Not necessary for this PR, but something to consider moving forward!